README.md

```diff
@@ -40,7 +40,7 @@ drat:::add("mrc-ide")
 install.packages("didehpc")
 ```
 
-or try the slightly experimental:
+For installation on R v3.5.0, use the slightly experimental:
```

Comment:
Why do you have this on 3.5.0? I think you're being mislead by R's terrible installation failure message perhaps ("not available for R version x.y.z") - both methods should work fine for all current R versions I believe